src/ray/gcs/client_test.cc

```diff
@@ -256,47 +259,116 @@ TEST_F(TestGcsWithAsio, TestLogAppendAt) {
   TestLogAppendAt(job_id_, client_);
 }
 
-void TestDeleteKeysFromLog(const JobID &job_id,
-                           std::shared_ptr<gcs::AsyncGcsClient> client,
-                           std::vector<std::shared_ptr<ObjectTableDataT>> &data_vector) {
+void TestSet(const JobID &job_id, std::shared_ptr<gcs::AsyncGcsClient> client) {
+  // Append some entries to the log at an object ID.
```

Comment:
Since the structure is changed to Set, this comment should be changed correspondingly. Append -> Add, Log -> Set.

Reply:
Thanks!